specification/offazurespringboot/resource-manager/Microsoft.OffAzureSpringBoot/preview/2023-01-01-preview/springbootdiscovery.json

```diff
@@ -824,32 +809,82 @@
           },
           {
             "name": "springbootapps",
-            "description": "Update a springbootapps payload.",
+            "description": "Create a springbootapps payload.",
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/springbootappsPatch"
+              "$ref": "#/definitions/springbootappsModel"
             }
           }
```

Comment:
>Creates [](http://example.com/codeflow?start=28&length=7)

PUT 200 applies to both create and update

Reply:
added

Reply:
I don't see any changes. Did you forget to push?

Reply:
@mentat9  I'm not very sure why it can't be seen in this pr, but in my branch can see the changes:[https://github.com/Azure/azure-rest-api-specs/commit/92d912245b7423b6683fb1f11f0cae115108cd08#diff-2610996d04da0dc7e5afb044e3eea049b546a5cf7f72bed525f17a6e4c98305f](url)